### PR TITLE
Create multiple templates recipe

### PR DESCRIPTION
**Pull request by jbleyleSF**

### CODE REVIEW

File: src/modules/recipe/miscMultipleTemplates/miscMultipleTemplates.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { LightningElement } from 'lwc';
+import templateOne from './templateOne.html';
+import templateTwo from './templateTwo.html';
+
+export default class MiscMultipleTemplates extends LightningElement {
+    templateOne = true;
```
Comment (muenzpraeger) [inline src/modules/recipe/miscMultipleTemplates/miscMultipleTemplates.js]: The property should be different than the imported template name. Like `showTemplateOne`.

File: src/modules/recipe/miscMultipleTemplates/miscMultipleTemplates.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+    render() {
+        return this.templateOne ? templateOne : templateTwo;
+    }
+
+    switchTemplate() {
+        this.templateOne = this.templateOne === true ? false : true;
```
Comment (muenzpraeger) [inline src/modules/recipe/miscMultipleTemplates/miscMultipleTemplates.js]: Simplify.

`this.showTemplateOne = !this.showTemplateOne`

**Comment by muenzpraeger** [comment]
@jbleyleSF What's the status of this? FYI @slychika @pozil

**Comment by jbleyleSF** [comment]
@muenzpraeger I'll finish it up this week and add the similar recipe to lwc-recipes.

**Pull request merged**